studio/data/analytics/infra-monitoring-query.ts

```diff
@@ -78,6 +78,7 @@ export const useInfraMonitoringQuery = <TData = InfraMonitoringData>(
           }),
         } as TData
       },
+      staleTime: 1000 * 60, // default good for a minute
```

Comment:
True, this can be improved further, that was just to avoid the constant re-requesting, we can cache even more, but didn't want to spend more time on this for now